reconcile/test/test_utils_mr_clusters_updates.py

```diff
@@ -0,0 +1,84 @@
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+from ruamel import yaml
+
+import reconcile.utils.mr.clusters_updates as sut
+
+from .fixtures import Fixtures
+
+fxt = Fixtures('clusters')
+
+
+class TrivialClustersUpdates(sut.CreateClustersUpdates):
+    # pylint: disable=super-init-not-called
+    def __init__(self, clusters_updates):
+        self.cancelled = False
+        self.message = ''
+        self.clusters_updates = clusters_updates
+        self.branch = 'abranch'
+        self.main_branch = 'main'
+
+    def cancel(self, message):
+        self.cancelled = True
+        self.message = message
+
+
+class TestProcess(TestCase):
+    def setUp(self):
+        self.clusters = [
+            fxt.get_anymarkup('cluster1.yml')
+        ]
+        self.raw_clusters = fxt.get('cluster1.yml')
+
+    def test_no_changes(self):
+        cli = MagicMock()
```

Comment:
Does it make sense to `create_autospec` here to catch cases where we're calling methods that don't even exist, etc? I won't comment on each instance of this.